src/web/mjs/connectors/BilibiliManhua.mjs

```diff
@@ -9,8 +9,15 @@ export default class BilibiliManhua extends Connector {
         super.label = '哔哩哔哩 漫画 (Bilibili Manhua)';
         this.tags = [ 'manga', 'webtoon', 'chinese' ];
         this.url = 'https://manga.bilibili.com';
-        this.lang = 'cn';
-
+        this.token_expires_at = -1;
+        this.auth = {
+            accessToken : '',
+            refreshToken : '',
+        };
+        this.links = {
+            login: 'https://passport.bilibili.com/login'
+        };
+        //default config doesnt need language switch, its fixed to CN
         this.config = {
```

Comment:
May add `language` property with `input: 'disabled'` and value set to `cn`.
With this all the ternary checks for `this.config.language` could be removed.
Disadvantage would be that this is shown in settings...

Reply:
I'll do that.